File: day2/trails/Q3.java
import java.util.Scanner;

public class Q3{
    public static void main(String[] args) {
        System.out.println("Enter the number");
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        if (n==0){
            System.out.println("The number is zero");
        }else if(n>0){
            if (n%2==0){
                System.out.println("The number is positive even");
            }
            else{
                System.out.println("The number is positive odd");
            }
        }
        else{
            if (n%2==0){
                System.out.println("The number is negative even");
            }
            else{
                System.out.println("The number is negative odd");
            }
        }

        sc.close();
    }
}

File: day2/trails/Q3.py
# write a program that takes an integer input from the user and categorizes the number as:
# - positive even
# - positive odd
# - negative even
# - negative odd
# - zero

num = int(input("Enter a number: "))

if num == 0:
    print(f"{num} is zero")
elif num > 0:
    if num % 2 == 0:
        print(f"{num} is positive even")
    else:
        print(f"{num} is positive odd")
else:
    if num % 2 == 0:
        print(f"{num} is negative even")
    else:
        print(f"{num} is negative odd")